### PR TITLE
Release v0.12.0

### LINKED ISSUE #141: Release 0.12
We've got several new features and some bug fixes in. I think it's about time for 0.12.

@ianobermiller: Anything else you think we should get in beforehand? If not will release today.

### LINKED ISSUE #145: Where is Radium.getState?
I do

````
    var Radium = require('radium');
````

but there is no Radium.getState

### PR DESCRIPTION
**Pull request by ianobermiller**
closes #141
closes #145

### CODE REVIEW

File: modules/resolve-styles.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ var getState = require('./get-state');
 var prefix = require('./prefix');
 
 var ExecutionEnvironment = require('exenv');
-var React = require('react/addons');
```
Comment (ianobermiller) [inline modules/resolve-styles.js]: My bad! This was causing us to suck react into the dist build!

**Comment by awkaiser** [comment]
@ianobermiller the **package.json** should have `"main": "modules/index.js"` instead of `"main": "lib/index.js"`

**Comment by awkaiser** [comment]
Ah, I see, the Babel transpiled modules end up in `lib` via `npm run lib`. Nevermind. :)

Giving this a whirl...

**Comment by awkaiser** [comment]
@ianobermiller & @alexlande: [As supported by](https://github.com/FormidableLabs/radium/pull/144#issuecomment-102191039) @doctyper, why not use [NPM prepublish + **.npmignore**](http://mammal.io/articles/using-es6-today/)? IMHO, NPM packages should be immediately usable via the entry script given set as `main` in **package.json**.

**Comment by doctyper** [comment]
Agreed, it would be great if you could get that and the [`rimraf`](https://github.com/FormidableLabs/radium/pull/144#discussion_r30285056) change in 0.12.

**Comment by ianobermiller** [comment]
@awkaiser I thought that was what we were doing, I guess not! Yes, I think prepublish is the way to go.

**Pull request merged**